From 1f17178e1c37539deff6d5a56b4b51a0c08fb68d Mon Sep 17 00:00:00 2001
From: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Date: Thu, 15 Feb 2024 18:31:10 +0100
Subject: [PATCH 0908/1141] arm64: dts: st: enable cryp1 on stm32mp257f-dk
 board

Set STM32 CRYP peripheral enabled.

Signed-off-by: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Change-Id: I4f6e4c22da90150faad405213b896187e75604d0
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/360584
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Yann GAUTIER <yann.gautier@foss.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
---
 arch/arm64/boot/dts/st/stm32mp257f-dk.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/st/stm32mp257f-dk.dts b/arch/arm64/boot/dts/st/stm32mp257f-dk.dts
index f7702bd821ba..0c0f08a3a77c 100644
--- a/arch/arm64/boot/dts/st/stm32mp257f-dk.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-dk.dts
@@ -185,6 +185,10 @@ &crc {
 	status = "okay";
 };
 
+&cryp1 {
+	status = "okay";
+};
+
 &csi2host {
 	vdd-supply =  <&scmi_vddcore>;
 	vdda18-supply = <&scmi_v1v8>;
-- 
2.39.2

